Sources/Basic/SortedArray.swift

```diff
@@ -1,12 +1,12 @@
 /*
  This source file is part of the Swift.org open source project
-
+ 
```

Comment:
Avoid running the re-indent for files but you can do that in your part of code. (select and re-indent)